projects/plugins/inspect/.phpcs.dir.phpcompatibility.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<ruleset>
+	<rule ref="PHPCompatibility.Constants.NewConstants">
```

Comment:
I'm wary of ignoring all of `PHPCompatibility.Constants.NewConstants` and `PHPCompatibility.Operators.NewOperators`. If there are new constants or operators added in 8.0+, we'd still want those warnings.

`PHPCompatibility.Syntax.NewFunctionCallTrailingComma` is probably ok, from the name this sniff seems likely to only apply to the one feature added in 7.3. OTOH, it would be just as easy to follow the rest and specify `PHPCompatibility.Syntax.NewFunctionCallTrailingComma.FoundInFunctionCall` fully for that too.

Reply:
Made #31711 that should make this sort of thing easier to get right.

Reply:
That got merged, we should use it here now.

Reply:
Done, thank you!